package.json

```diff
@@ -12,8 +12,9 @@
     "type": "git",
     "url": "https://github.com/SAP/techne.git"
   },
-  "author": "Hybris Software",
-  "license": "MIT",
+  "homepage": "http://techne.yaas.io/",
```

Comment:
@jensru I'm speaking with @taifunhaeberle  and this doesn't seem to be correct.  Can you confirm wether this is the correct update that we should have?